content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -3046,12 +3050,12 @@ Creates a new [Customer](#customer) user type.
 
 #### Request
 
-| Parameter | Required | Data type | Notes                                |
-| --------- | -------- | --------- | ------------------------------------ |
-| `name`    | No       | `string`  |                                      |
-| `email`   | No       | `string`  |                                      |
-| `avatar`  | No       | `string`  | URL of the Customer's avatar         |
-| `fields`  | No       | `object`  | A map in the `"key": "value"` format |
+| Parameter        | Required | Data type  | Notes                                                                  |
+| ---------------- | -------- | ---------- | ---------------------------------------------------------------------- |
+| `name`           | No       | `string`   |                                                                        |
+| `email`          | No       | `string`   |                                                                        |
+| `avatar`         | No       | `string`   | URL of the Customer's avatar                                           |
+| `session_fields` | No       | `[]object` | An array of custom object-enclosed `key:value`s. Item order sensitive. |
```

Comment:
```suggestion
| `session_fields` | No       | `[]object` | An array of custom object-enclosed `key:value` pairs. Respects the order of items. |
```

Reply:
Item order sensitive - this cluster is a bit confusing when you read it for the first time